packages/package_info_plus/package_info_plus_windows/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 
 dependencies:
   package_info_plus_platform_interface: ^1.0.2
-  ffi: ^1.0.0
+  ffi: ^2.0.0
```

Comment:
The latest version is `2.0.1`.

Reply:
Fixed https://github.com/fluttercommunity/plus_plugins/pull/932/files#diff-b4791c5bcf19c39466a3745038d1428342ae76b0c586f68d5142df5fe9cc0810R13